website/docs/cloud-docs/workspaces/dynamic-provider-credentials/hcp-configuration.mdx

```diff
@@ -0,0 +1,110 @@
+| `TFC_HCP_PROVIDER_AUTH`                     | `true`                                                                                               | Requires **v1.15.1** or later if self-managing agents. Must be present and set to `true`, or HCP Terraform will not attempt to use dynamic credentials to authenticate to HCP.                                                                                                                                           |
+| `TFC_HCP_RUN_PROVIDER_RESOURCE_NAME`        | The resource name of the workload identity provider that will be used to assume the service principal| Requires **v1.15.1** or later if self-managing agents. Optional if `PLAN_PROVIDER_RESOURCE_NAME` and `APPLY_PROVIDER_RESOURCE_NAME` are both provided. These variables are described [below](/terraform/cloud-docs/workspaces/dynamic-provider-credentials/hcp-configuration#optional-environment-variables) |
+
+### Optional Environment Variables
+You may need to set these variables, depending on your use case.
+
+| Variable                                      | Value                                                                                                                                                                                                                                                                      | Notes                                                                                                                                     |
+|-----------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| `TFC_HCP_WORKLOAD_IDENTITY_AUDIENCE`          | Will be used as the `aud` claim for the identity token. Defaults to the provider resource name for the current run phase that was derived from the values you provided for `RUN_PROVIDER_RESOURCE_NAME`, `PLAN_PROVIDER_RESOURCE_NAME`, and `APPLY_PROVIDER_RESOURCE_NAME` | Requires **v1.15.1** or later if self-managing agents. This is one of the default `aud` formats that HCP accepts.                          |
+| `TFC_HCP_PLAN_PROVIDER_RESOURCE_NAME`         | The resource name of the workload identity provider that will be used to assume the service principal for the plan phase of a run.                                                                                                                                         | Requires **v1.15.1** or later if self-managing agents. Will fall back to the value of `RUN_PROVIDER_RESOURCE_NAME` if not provided.        |
```

Comment:
I wasn't quite sure what "will be used to assume" meant here. I tried to make this a bit more clear:
```suggestion
| `TFC_HCP_PLAN_PROVIDER_RESOURCE_NAME`         | The resource name of the workload identity provider that HCP Terraform assumes as the service principal for the plan phase of a run.                                                                                                                                         | Requires **v1.15.1** or later if self-managing agents. If you do not provide a value, defaults to `RUN_PROVIDER_RESOURCE_NAME` value.        |
```

Reply:
"assumes" as in "assumes the role of the service principal". Good catch though, I'll clarify!